solr/core/src/test/org/apache/solr/search/TestSolr4Spatial.java

```diff
@@ -210,7 +210,8 @@ public void testIntersectFilter() throws Exception {
   public void checkResultFormat() {
     // Check input and output format is the same
     String IN = "89.9,-130"; // lat,lon
```

Comment:
How about combining the two variables?
```suggestion
    final String IN_OUT = "89.9,-130"; // lat,lon
```